Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2567,11 +2567,12 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
         return;
 
 #if PLATFORM(MAC)
-    NSString *urlAbsoluteString = [url absoluteString];
-    NSArray *types = @[ NSPasteboardTypeString, NSPasteboardTypeHTML ];
-    NSArray *items = @[ [urlAbsoluteString dataUsingEncoding:NSUTF8StringEncoding], [urlAbsoluteString dataUsingEncoding:NSUTF8StringEncoding] ];
-
-    writeItemsToPasteboard(NSPasteboardNameGeneral, items, types);
+    RetainPtr urlData = [[url absoluteString] dataUsingEncoding:NSUTF8StringEncoding];
+    Vector<PasteboardItem> pasteboardItems {
+        { urlData, NSPasteboardTypeString },
+        { urlData, NSPasteboardTypeHTML },
```

Comment:
Would this write HTML data for a copied URL? I think we'd probably either just want the plain string, or use `NSPasteboardTypeURL`

Reply:
Yeah this seemed odd to be, but I didn't question it since we migrated this from legacy PDF plugin.

I do think if we advertise a second data type here then `NSPasteboardTypeURL` is the better fit. I'll make sure that change doesn't break anything and then update this line.